app/src/main/java/com/pajato/android/gamechat/main/NavigationManager.java

```diff
@@ -196,17 +223,37 @@ private void enableDrawerMenuGroup(final Activity activity) {
         }
     }
 
+    /** Access one of four possible alternate account icon ids. */
+    private int getAlternateAccountIconId(final int index) {
```

Comment:
fwiw, I have begun preferring to code switches like these using return statements, like:
```switch (foo) {```
``` case 0: return foo1Result;```
```case 1: return  foo2Result;```
```...```
```default: return -1```
```}```
in order to achieve conciseness.